google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -5179,4 +5181,58 @@ public void testCreateExternalTableWithReferenceFileSchemaParquet() {
     boolean success = bigquery.delete(tableId);
     assertEquals(true, success);
   }
+
+  @Test
+  public void testCloneTableCopyJob() throws InterruptedException {
+    String sourceTableName = "test_copy_job_base_table";
+    String ddlTableName = TABLE_ID_DDL.getTable();
+    String cloneTableName = String.format("test_clone_table");
+    // Create source table with some data in it
+    String ddlQuery =
+        String.format(
+            "CREATE OR REPLACE TABLE %s ("
+                + "TimestampField TIMESTAMP OPTIONS(description='TimestampDescription'), "
+                + "StringField STRING OPTIONS(description='StringDescription'), "
+                + "BooleanField BOOLEAN OPTIONS(description='BooleanDescription') "
+                + ") AS SELECT * FROM %s",
+            sourceTableName, ddlTableName);
+    QueryJobConfiguration ddlConfig =
+        QueryJobConfiguration.newBuilder(ddlQuery).setDefaultDataset(DatasetId.of(DATASET)).build();
+    TableId sourceTableId = TableId.of(DATASET, sourceTableName);
+    TableResult result = bigquery.query(ddlConfig);
+    assertEquals(DDL_TABLE_SCHEMA, result.getSchema());
+    Table remoteTable = bigquery.getTable(DATASET, sourceTableName);
+    assertNotNull(remoteTable);
+
+    // Create clone table using source table as the base table
+    TableId cloneTableId = TableId.of(DATASET, cloneTableName);
+    CopyJobConfiguration cloneConfiguration =
+        CopyJobConfiguration.newBuilder(cloneTableId, sourceTableId)
+            .setOperationType("CLONE")
+            .build();
+    Job createdJob = bigquery.create(JobInfo.of(cloneConfiguration));
+    CopyJobConfiguration createdConfiguration = createdJob.getConfiguration();
+    assertNotNull(createdConfiguration.getSourceTables());
+    assertNotNull(createdConfiguration.getOperationType());
+    assertNotNull(createdConfiguration.getDestinationTable());
+    Job completedJob = createdJob.waitFor();
+    assertNull(completedJob.getStatus().getError());
+
+    Table cloneTable = bigquery.getTable(DATASET, cloneTableName);
+    assertNotNull(cloneTable);
+    System.out.println(cloneTable.getDefinition().getType());
```

Comment:
Guessing the System.out.println lines in here are stale debug statements that survived?

Reply:
Yes